src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableFieldWriter.class.php

```diff
@@ -152,18 +152,24 @@ protected function PostSaveHook(&$oFields, &$oPostTable)
 
         $this->_oField = $this->adaptFieldAndRelatedTables($oPostTable->sqlData);
 
-        $this->getAutoclassesCacheWarmer()->updateTableById($oPostTable->sqlData['cms_tbl_conf_id']);
+        if (\array_key_exists('cms_tbl_conf_id', $oPostTable->sqlData)) {
+            $this->getAutoclassesCacheWarmer()->updateTableById($oPostTable->sqlData['cms_tbl_conf_id']);
+        }
     }
 
     /**
      * @param array $postData
      *
-     * @return TCMSField - the new definition after changes
+     * @return TCMSField|null - the new definition after changes; or null for small changes (without type)
      *
      * @throws DBALException
      */
-    protected function adaptFieldAndRelatedTables(array $postData): TCMSField
+    protected function adaptFieldAndRelatedTables(array $postData): ?TCMSField
```

Comment:
This is a BC break that is not allowed in a fix release. Can we instead only call this method if the requirements are met?

Reply:
ok